### PR TITLE
 Movies page/Feature/New page #29 

### PR DESCRIPTION
**Pull request by Kirill9m**
# Description:
New page where we can display a list of all movies and other features
Issue #29

# Type of change:
- New page is created
- Copied the design from the main page (@jennymakki footer, buttons and initial design from @alfredbrannare) 
- A new design for the header can be used as a temporary option since there hasn’t been one yet. (So that the card doesn’t hang while I’m waiting)
- Updated vite.config.js file with the new page information

# How its looks like?
<details><summary>Click me</summary>
<p>

image

</p>
</details> 

# Additional information (screenshot, video, code examples):
The planned time was 1 hour, but it took 2 hours due to working on the design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +36,9 @@
           </li>
         </ul>
       </nav>
-      <a class="header__bookTicket" aria-label="Book tickets now">BOKA BILJETT</a>
+      <a class="header__bookTicket" aria-label="Book tickets now">
+        <button class="primary_button">BOKA BILJET</button>
```
Comment (Paajt) [inline index.html:40]: Just a typo on "Biljett".

**Review by Paajt — APPROVED** [review]
Looks great, good job! Both the code and your images provided in PR 👌

File: index.html (hunk 1 above)
Comment (Kirill9m) [reply index.html]: Klart!

File: images/kino-logo.png
Comment (jennymakki) [inline images/kino-logo.png:1]: Screenshot 2024-12-22 at 19 54 20
I know this header is temporary but here is the updated logo.

**Review by jennymakki — APPROVED** [review]
Great job with everything!

File: images/kino-logo.png
Comment (Kirill9m) [reply images/kino-logo.png:1]: Fixed!

**Pull request merged**